src/test/java/com/smartbear/readyapi/client/DataGenDataSourceTest.java

```diff
@@ -240,6 +242,38 @@ public void buildsRecipeWithDataSourceTestStepWithComputerAddressDataGenWithMac4
     }
 
     @Test
+    public void buildsRecipeWithDataSourceTestStepWithStateNameDataGenWithFullNames() throws Exception {
+        TestRecipe recipe = newTestRecipe()
+                .addStep(dataGenDataSource()
+                        .withProperty(
+                                stateNameTypeProperty("property1")
+                                        .withFullNames()
```

Comment:
I guess you could abbreviate this to just fullStateNameTypeProperty("property1")